src/utils/daos.c

```diff
@@ -927,6 +1014,23 @@ call_dfuse_ioctl(char *path, struct dfuse_il_reply *reply)
 	return 0;
 }
 
+static int
+fs_op_hdlr(struct cmd_args_s *ap)
+{
+	int rc = 0;
+	enum fs_op op;
```

Comment:
checkpatch should have complained here. need an empty line after functiona variable declarations.